### ISSUE TITLE
docs/_static/ not present in PyPI tarballs

### ISSUE THREAD

**Issue by sbraz** [erikrose/more-itertools#218]
Hi,
building the docs from the PyPI release result in the following warning:
```
copying static files... WARNING: html_static_path entry '/var/tmp/portage/dev-python/more-itertools-4.2.0/work/more-itertools-4.2.0/docs/_static' does not exist                                                  
```
You probablby need to add this to MANIFEST.in

**Comment by bbayles** [comment]
Ah, you're right - the `_static` directory is not present in the most recent release, but it it there in `master.

**Comment by sbraz** [comment]
Oh so it wasn't a problem of it not being included; it simply wasn't present in v4.2.0. I guess the next release will be fine.

**Comment by bbayles** [comment]
That should do it - thanks!
